vlsi/example-vlsi-sky130

```diff
@@ -32,35 +32,15 @@ def example_tool_settings(x: hammer_vlsi.HammerTool) -> bool:
         x.append('''
 # TODO
 # Place custom TCL here
+# only route in met1 to met4
```

Comment:
is `met1` equal to layer 2?

Reply:
yes, `li1` (local interconnect) is layer 1, making the remaining layers off by 1 from metal names. I can expand this comment to clarify that.

Reply:
K. I think for Innovus, it also understands if you put the layer name rather than number in that command too.